firmware/application/apps/ui_recon.cpp

```diff
@@ -1339,25 +1338,26 @@ size_t ReconView::change_mode(freqman_index_t new_mod) {
     field_mode.on_change = [this](size_t, OptionsField::value_t) {};
     field_bw.on_change = [this](size_t, OptionsField::value_t) {};
     recon_stop_recording();
-    if (new_mod != SPEC_MODULATION) {
-        remove_children({record_view});
```

Comment:
In my tests it was needed to  remove and add back the children to correctly swap from audio to raw.
Maybe I was wrong, have you tried to record in both modes ?

Reply:
The children are still always being removed. The only change I was a little unsure about was the location the “add_child”, but it should be fine.

Reply:
The diff layout is terrible, but remove happens before stopping the audio and swapping the RecordView mode. It’s re-added at the end.

Reply:
Ho my bad you're right. I should have checked the full file before, and yes, that diff IS terrible X-D